src/buildercore/fastly/vcl/original-host.vcl

```diff
@@ -1 +1,3 @@
-set req.http.X-Forwarded-Host = req.http.host;
+if (!req.http.Fastly-FF) {
```

Comment:
needs a comment explaining the reason for the check (which should end up in the generated VCL)